lib/tasks/get-space-data.js

```diff
@@ -107,6 +107,29 @@ export default function getFullSourceSpace ({
       }),
       skip: () => skipContent
     },
+    {
+      title: 'Fetching tags data',
+      task: wrapTask((ctx, task) => {
+        return pagedGet({source: ctx.environment, method: 'getTags'})
+            .then(extractItems)
+            .then((items) => {
+              ctx.data.tags = items
+            })
+            .catch(e => {
+              console.log(`Caught error: ${e}`)
+              // not all spaces have tags enabled
+              ctx.data.tags = []
+              return
+            })
+      }),
+      skip: (ctx) => {
```

Comment:
Re: my comment above, then we may not need to skip fetching tags when `skipContent` is used.
For the cda token, maybe we can instead skip if cdaClient exists as it was done [here](https://github.com/contentful/contentful-export/blob/bee08ffddf5a3cdbfec4db4d6e0ec88ab4eb1314/lib/tasks/get-space-data.js#L78)?

Reply:
I agree with the above. But why not just skip fetching tags if using cda token?  WDYT?

```
skip: () => cdaToken
```

Reply:
If `cdaToken` exists and is accessible, I guess that's fine too. I assumed `cdaClient` was the only thing available in this file.